ann/ann.py

```diff
@@ -0,0 +1,113 @@
+# ============================================================================
+# Neural Network functions using TensorFlow and Keras
```

Comment:
Can you add a single line that explains that this code is for N features and 1 output?